modules/tinymce/src/core/main/ts/newline/InsertBlock.ts

```diff
@@ -1,4 +1,11 @@
-import { Arr, Obj, Optional, Optionals } from '@ephox/katamari';
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
```

Comment:
```suggestion
```
This looks like a merge commit that's gone bad, as this header is no longer needed.